Tone/core/context/ToneAudioNode.ts

```diff
@@ -376,3 +376,18 @@ export function disconnect(
 		srcNode.disconnect();
 	}
 }
+
+/**
+ * Connect the output of this node to the rest of the nodes in parallel.
+ * @param nodes One or more source nodes followed by one destination node
+ * @example
+ * const player = new Tone.Player("https://tonejs.github.io/audio/drum-samples/conga-rhythm.mp3");
+ * const player1 = new Tone.Player("https://tonejs.github.io/audio/drum-samples/conga-rhythm.mp3");
+ * const filter = new Tone.Filter("G5").toDestination();
+ * // connect nodes to a common destination
+ * fanIn(filter, player, player1);
+ */
+export function fanIn(...nodes: any[]): void {
```

Comment:
better to not use `any[]` here. Maybe `OutputNode[]` (even though it's not totally correct)

Reply:
Done – thanks for the feedback! See latest commit.